tests/e2e/cucumber/steps/ui/adminSettings.ts

```diff
@@ -300,3 +300,33 @@ When(
     }
   }
 )
+
+When(
+  /^"([^"]*)" deletes the following (user|users) using the (batch actions|context menu)$/,
+  async function (
+    this: World,
+    stepUser: string,
+    _: string,
+    actionType: string,
+    stepTable: DataTable
+  ): Promise<void> {
```

Comment:
I think we can do this in regex
```suggestion
When(
  /^"([^"]*)" deletes the following (?:user|users) using the (batch actions|context menu)$/,
  async function (
    this: World,
    stepUser: string,
    actionType: string,
    stepTable: DataTable
  ): Promise<void> {
```